File: Arrays/217leetcodeContainsDuplicate/217leetcode.py
# https://leetcode.com/problems/contains-duplicate/submissions/1728711397/


class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """

        s=set(nums)
        if len(nums)==len(s):
            return False
        return True
        
        # nums.sort()
        # for i in range(len(nums) - 1):
        #     if nums[i]==nums[i+1]:
        #         return True  
        # return False    

File: Arrays/217leetcodeContainsDuplicate/217leetcode.java
// https://leetcode.com/problems/contains-duplicate/submissions/1728711397/
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> sets=new HashSet<>();
        for(int i:nums){
            sets.add(i);
        }
        if(nums.length==sets.size()){
            return false;
        }
        return true;
    }
}